engine_modules/corporation/tasks.py

```diff
@@ -10,11 +12,28 @@ class CrashCorporationTask(ResolutionTask):
 	RESOLUTION_ORDER = 850
 
 	def run(self, game):
-		corporations_to_crash = game.corporation_set.filter(assets__lte=0)
+		corporations_to_crash = list(game.corporation_set.filter(assets__lte=0))
 
 		if not corporations_to_crash:
 			return
 
+		first_crash_phoenix = False
+		# Handle the case of phoenix part 1
+		for corporation in corporations_to_crash:
+			if corporation.base_corporation.phoenix == 1:
+				# avoid others crashed effects to appy on Phoenix
+				corporation.crash_turn = game.current_turn
```

Comment:
Yes, it is said in the rules that a crashed corporation is not affected by crashed effects. (to avoid the fact that the others crashed effect can revive them or kill them again for the phoenix)